libherokubuildpack/src/inventory/artifact.rs

```diff
@@ -4,6 +4,15 @@ use serde::{Deserialize, Serialize};
 use std::fmt::Display;
 use std::str::FromStr;
 
+/// Representation of a downloadable artifact such as a binary tarball.
+///
+/// An inventory is made up of multiple artifacts that have a version that
+/// can be compared to each other and a URL where the artifact can be downloaded.
```

Comment:
This is a bit inaccurate.. The inventory itself doesn't actually require the artifact version to be comparable -- the `resolve` and `resolve_partial` functions add those constraints (e.g. `Ord` and `PartialOrd`) on the `Version` type.

Reply:
I mean you're right in the sense that most consumers of this module will likely use the `resolve` and `resolve_partial` methods on the `Inventory` struct. But this is documentation for the `Artifact` struct, which is deliberately decoupled from any `ArtifactRequirement` constraints -- so I guess that's why it stood out to me in this context.